### PR TITLE
fix(UI): redirect user to his default page

### LINKED ISSUE #6548: Load the default page when user is already authenticated
Hi,

The _Default page_ setting works at user login (when user enters his username/password). Perfect.

However, it does not work when the browser is already authenticated, and user navigates to `http://server/centreon`. User is then not redirected and sees the _Home > Custom Views_ page.

Could it be possible to load the _Default page_ here too ?

Thank you very much :+1: 

Edit : issue still present in 19.10.10.

### PR DESCRIPTION
**Pull request by UrBnW**
Hi,

## Description

When a user is already logged-in, he will not be redirected to his default page in new tabs / windows.
This PR then redirects the user to his default page reaching Centreon portal, when already logged-in.

It then closes #6548

## Type of change

- [x] Patch fixing an issue (non-breaking change)

## Target serie

- [x] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Login to Centreon.
Open a new tab or window to your Centreon portal, and be sure to reach your own default page.

Edit : issue still present in 19.10.10.

### CODE REVIEW

File: www/index.php [hunk 1]
```diff
@@ -125,7 +125,11 @@
  */
 if (isset($_SESSION["centreon"])) {
     $centreon = &$_SESSION["centreon"];
-    header('Location: main.php');
+    $headerRedirection = "main.php";
+    if (isset($centreon->user->default_page) && $centreon->user->default_page != '') {
```
Comment (kduret) [inline www/index.php]:
```suggestion
    if (!empty($centreon->user->default_page)) {
```

**Comment by UrBnW** [comment]
Many thanks for your review @kduret 👍

**Comment by sc979** [comment]
Hi @UrBnW ,
And, sorry for the delay, but we're overwhelmed right now.
Your modifications are in the backlog under the reference MON-4954
Best regards,
sc979

**Comment by kduret** [comment]
I close this PR cause it has been fixed by following PRs : 
- https://github.com/centreon/centreon/pull/8581
- https://github.com/centreon/centreon/pull/8644

Thanks for contribution and sorry to not handle this PR earlier

**Comment by UrBnW** [comment]
Thank you @kduret.

**Pull request closed without merging**